test/extended/deployments/deployments.go

```diff
@@ -236,18 +237,28 @@ var _ = g.Describe("deploymentconfigs", func() {
 			o.Expect(config.Spec.Test).Should(o.BeTrue())
 
 			g.By("deploying a few more times")
+			// expression to match the name of the rolled deployment
+			deploymentVersionExp := regexp.MustCompile(`"deployment-test-(\d+)" has completed progressing`)
```

Comment:
@kargakis are the version numbers reliable (i+2) (iow. i don't need this regexp)